frontend/src/metabase/static-viz/components/XYChart/XYChart.tsx

```diff
@@ -5,7 +5,7 @@ import { GridRows } from "@visx/grid";
 import { Group } from "@visx/group";
 import { assoc } from "icepick";
 
-import { formatNumber } from "metabase/static-viz/lib/numbers";
+import { formatNumber } from "metabase/lib/formatting/numbers";
```

Comment:
Use the same format as the main app.

Note that I can't import from `metabase/lib/formatting` (without appending `/numbers`) since our Webpack setup doesn't seem to tree-shake the code resulting in static viz build failure, since we can't load certain modules in static viz.